### ISSUE TITLE
radio button btn-group checkedness doesn't get updated immediately

### ISSUE THREAD

**Issue by khannavidur** [twbs/bootstrap#18139]
Once a radio button is checked( clicked on ) ...     	console.log($('input: radio :checked').attr('id'));
doesn't change. But changes after the consecutive clicks.

A reproduction of the issue is @ https://jsfiddle.net/b9gbx069/2/

Check console for the irregular behavior  observed.

**Comment by twbs-lmvtfy** [comment]
Hi @khannavidur!

You appear to have posted a live example (https://fiddle.jshell.net/b9gbx069/1/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 53, column 5 thru column 70: Duplicate attribute ```` value ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by khannavidur** [comment]
Hi ... posted the old fiddle url .... https://jsfiddle.net/b9gbx069/2/ ...

**Comment by cvrebert** [comment]
@khannavidur In what browser and OS?

**Comment by khannavidur** [comment]
Chrome .... Ubuntu 14.0.4

**Comment by khannavidur** [comment]
Google Chrome Exact version - 46.0.2490.80 (64-bit)

**Comment by cvrebert** [comment]
X-Ref: #13850
Due to the way the event handling works, your handler needs to be at the document level if it wants the correct post-click checkedness. Example: https://jsfiddle.net/bp094393/7/
This bears documenting.

**Comment by khannavidur** [comment]
Thank you ... meantime I found a workaround with the label's ids ... but that was applicable in just my use case.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
